go.mod

```diff
@@ -2,132 +2,148 @@ module github.com/oasisprotocol/nexus
 
 go 1.19
 
-replace github.com/tendermint/tendermint => github.com/oasisprotocol/tendermint v0.34.21-oasis1
+replace ( // All from oasis-core
+	github.com/cometbft/cometbft => github.com/oasisprotocol/cometbft v0.37.2-oasis1
+
+	// v1.5.0 has broken uint parsing, use my commit with fixes instead until
+	// the maintainers merge my PR: https://github.com/spf13/cast/pull/144
+	github.com/spf13/cast => github.com/oasisprotocol/cast v0.0.0-20220606122631-eba453e69641
+	github.com/tendermint/tendermint => github.com/oasisprotocol/tendermint v0.34.21-oasis1
+
+	golang.org/x/crypto/curve25519 => github.com/oasisprotocol/curve25519-voi/primitives/x25519 v0.0.0-20210505121811-294cf0fbfb43
+	golang.org/x/crypto/ed25519 => github.com/oasisprotocol/curve25519-voi/primitives/ed25519 v0.0.0-20210505121811-294cf0fbfb43
+)
 
 require (
-	github.com/dgraph-io/ristretto v0.1.0
+	github.com/dgraph-io/ristretto v0.1.1
 	github.com/ethereum/go-ethereum v1.12.1
 	github.com/go-chi/chi/v5 v5.0.7
 	github.com/go-kit/log v0.2.1
 	github.com/golang-migrate/migrate/v4 v4.15.2
 	github.com/google/uuid v1.3.0
 	github.com/jackc/pgx/v5 v5.3.1
 	github.com/knadh/koanf v1.4.1
-	github.com/oasisprotocol/oasis-core/go v0.2202.11
-	github.com/oasisprotocol/oasis-sdk/client-sdk/go v0.6.0
+	github.com/oasisprotocol/oasis-core/go v0.2300.0
```

Comment:
tag exists now, could you update "bump default oasis-core to 23.0" commit message

Reply:
updated

Reply:
There is v0.2300.2 now already :-)